src/index.js

```diff
@@ -7,12 +7,13 @@ import modifyStatus from './status-module';
 import {
   getTasksLocalStorage,
   addTask,
-  setTasksLocalStorage,
   modifyTask,
   deleteSelectedTask,
```

Comment:
- [ ] **Required:** The test should not break the functionality of your app. At the moment, the delete button/function is not working correctly. Please fix this bug.